chromium_src/chrome/browser/ui/tabs/tab_strip_model.cc

```diff
@@ -0,0 +1,12 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/tabs/tab_strip_model.h"
+
+#include "chrome/browser/ui/views/tabs/tab_drag_controller.h"
+
+#define TabDragController TabDragControllerChromium
```

Comment:
I'm confused because I don't even see `TabDragController` referenced in tab_strip_model.cc

Reply:
cc: @cdesouza-chromium

Reply:
I think I was behind on some commits, I see now, but why does this have to be the original `TabDragController`? Shouldn't everything use our subclass including the PassKey?

Reply:
The issue is that `PassKey<TabDragController>` can't be used for `PassKey<TabDragControllerChromium>` and vice versa, so you have to match that as well.